jbpm-benchmarks/jbpm-performance-tests-jmh/src/main/java/org/jbpm/test/performance/scenario/load/services/QueryProcessesAndTasksByVariables.java

```diff
@@ -127,32 +123,28 @@ public void init() throws Exception {
 
         // Update task variables twice
         taskVariables.forEach( (k, v) -> taskVariables.replace(k, counter.getAndIncrement()));
-        processIds.forEach( id -> {
-            runtimeDataService.getTasksByProcessInstanceId(id).forEach( taskId -> {
-                UserTaskInstanceDesc task = runtimeDataService.getTaskById(taskId);
-                Map<String, Object> newInputTask = new HashMap<>();
-                userTaskService.getTaskInputContentByTaskId(taskId).keySet().stream()
-                        .filter(o -> o.contains("task_var_in"))
-                        .forEach( key -> {
-                            newInputTask.put(key, taskVariables.get(key));
-                            userTaskService.updateTask(taskId, "salaboy", task, newInputTask, null);
```

Comment:
true - thanks for that.
I've already changed the perf test to use both input/output vars task instead in a later PR I have prepared for this (was because of starting all processes in a multi-thread + using a separate junit test to create the scenario). Though It's not raised yet